### PR TITLE
Fix typos, etc. in glossary (Closes #1477)

### LINKED ISSUE #1477: Issues with the glossary
## Where to find the issue
https://www.coronawarn.app/de/faq/ - Glossar

### B
- **Business rules**:  Change "Impfzertifikate für COVID19" to "Impfzertifikate für COVID-19"

### D
- **DCC**: Change "das digitale COVID Zertifikat" to "das digitale COVID-Zertifikat"

### G
- **Genesenen-Zertifikat**: 
	1. Change "eine Covid-19-Erkrankung" to "eine COVID-19-Erkrankung"
	2. Why "(PCR-) Test"? I though recovery certificates are only issued for people who can prove their infection with a PCR test, not with a RAT.
- **Gültigkeit eines Zertifikats**: 
	1. Fix double "hat"
	2. Remove the * from `*In*`
	3. Change "in DE 1" to "in Deutschland 1"
	4. Add missing "."

### P
- **Prüf-Aapp / Prüfen eines Zertifikats / Verifizieren eines Zertifikats**:
	1. Change "Prüf-Aapp" to "Prüf-App"
	2. Fix the highlighting `*muss*`
	3. Remove "durch"
	4. Change "CovPass Check App" to "CovPassCheck-App"
	
### S
- **Signatur (/Signierung eines Zertifikats)**:
	1. Remove the "/"
	2. Change "Dokumentunterzeichnet wurde" to "Dokument unterzeichnet wurde" 
	3. Remove some "." too much 

### T
- **Testzertifikat**
	1. Change "Das Impfzertifikat" to "Das Testzertifikat"
	2. Change "mit einem Covid-19 PCR-Test" to "einen COVID-19 PCR-Test"
		- Why only PCR-Tests? You can also request test certificates for RAT in the CWA.

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR fixes some issues with the text of the glossary.
You can find more informations in #1477.

---

**Closes #1477**

### CODE REVIEW

**Comment by Ein-Tim** [comment]
FYI @svengabr

File: src/data/faq_de.json [hunk 1]
```diff
@@ -2030,11 +2030,11 @@
...
+                    "description": "Wir unterscheiden grundsätzlich<ul><li>die Gültigkeit des digitalen Zertifikats (die technische Gültigkeit) und</li><li>die Gültigkeit des Impfschutzes (die fachliche Gültigkeit).</li></ul>Das technische Zertifikat hat ein Ausstellungsdatum und ein technisches Ablaufdatum (so wie auch Ihr Reisepass und PA). Die Gültigkeit des Zertifikats ist 1 Jahr nach Ausstellung. <br/><br/> In dem Zertifikat werden Daten transportiert, in diesem Falle Informationen zur Impfung. Diese Impfung fand an einem Datum X statt und gilt ab dann (nach aktuellem Stand der wissenschaftlichen Erkenntnisse) z.B. in Deutschland 1 Jahr. Danach ist der Impfschutz erloschen. Die Dauer des Impfschutzes bezeichnen wir als 'fachliche' Gültigkeit. Die fachliche Gültigkeit Impfung zählt also ab dem letzten Impfdatum (Datum X). <br/><br/> Die Check App prüft zunächst die technische Gültigkeit des Zertifikats und dann die inhaltliche/fachliche Gültigkeit. <br/><br/> Unerheblich davon, welche der beiden Gültigkeiten zuerst abläuft, in beiden Fällen wird das gesamte Zertifikat bei der Verifikation mit der Check App als „ungültig“ abgelehnt."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2037]: z.B. => `z.&nbsp;B.`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Just one additional comment about the abbreviation z.B. missing a non-breaking space.

File: src/data/faq_de.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Thank you! Done in https://github.com/corona-warn-app/cwa-website/pull/1478/commits/f57cd216d8115f4d501c70ca6aee5e1327b205c4

**Pull request merged**